feature/qos/otg_tests/egfess_traffic_classification_and_rewrite_test

```diff
@@ -0,0 +1,190 @@
+    *   Capture packets on the ATE's ingress interface to verify packet marking according to the marking table.
+    *   Analyze traffic flows to confirm that no packets are dropped on the DUT.
+
+### DP-1.17.5 Egress Classification and rewrite of IPoMPLSoGRE traffic with pop action
+*   Configuration:
+    *   Configure Static MPLS LSP with MPLS pop and IPv4 forward actions for a specific labels range (100020-100100).
+    *   Configure decapsulation rules for IPv4-DST-DECAP/32
+*   Traffic:
+    *   Generate IPoMPLSoGRE traffic from ATE Port 1 with labels between 100020 and 1000100
+*   Verfication:
+    *   Monitor telemetry on the DUT to verify that packets re being scehduled for transmission into correct forwarding groups.
+    *   Capture packets on the ATE's ingress interface to verify packet marking according to the marking table.
+    *   Analyze traffic flows to confirm that no packets are dropped on the DUT.
+
+### DP-1.17.7 Egress Classification and rewrite of IPv6oMPLSoGRE traffic with pop action
+*   Configuration:
+    *   Configure Static MPLS LSP with MPLS pop and IPv6 forward actions for a specific labels range (100020-100100).
+    *    Configure decapsulation rules for IPv6-DST-DECAP/12 
+*   Traffic:
+    *   Generate IPv6oMPLSoGRE traffic from ATE Port 1 with labels between 100020 and 1000100
+*   Verfication:
+    *   Monitor telemetry on the DUT to verify that packets re being scehduled for transmission into correct forwarding groups.
+    *   Capture packets on the ATE's ingress interface to verify packet marking according to the marking table.
+    *   Analyze traffic flows to confirm that no packets are dropped on the DUT.
+
+### DP-1.17.8 Egress Classification and rewrite of IPoGRE traffic with pop action
+*   Configuration:
+    *    Configure decapsulation rules for IPv4-DST-DECAP/32
+*   Traffic:
+    *   Generate IPoGRE traffic from ATE Port 1 with IP payload dest reachable via ATE:Port2
+*   Verfication:
+    *   Monitor telemetry on the DUT to verify that packets re being scehduled for transmission into correct forwarding groups.
+    *   Capture packets on the ATE's ingress interface to verify packet marking according to the marking table.
+    *   Analyze traffic flows to confirm that no packets are dropped on the DUT.
+
+### DP-1.17.9 Egress Classification and rewrite of IPv6oGRE traffic with pop action
+*   Configuration:
+    *   Configure decapsulation rules for  IPv6-DST-DECAP/12 
+*   Traffic:
+    *   Generate IPv6oGRE traffic from ATE Port 1 with payload reachable via ATE:Port2
+*   Verfication:
+    *   Monitor telemetry on the DUT to verify that packets re being scehduled for transmission into correct forwarding groups.
+    *   Capture packets on the ATE's ingress interface to verify packet marking according to the marking table.
+    *   Analyze traffic flows to confirm that no packets are dropped on the DUT.
```

Comment:
Please review [PF-1.3 Policy-based IPv4 GRE Decapsulation](https://github.com/openconfig/featureprofiles/blob/ac98ad2d2a57b460ab6584d4b26f99145a04ba7e/feature/policy_forwarding/encapsulation/otg_tests/decap_gre_ipv4/README.md?plain=1)

I think this test overlaps with PF-1.3, but adds some packet marking rules.  In order to specify exactly what configuration is needed (add structure to expose how this qos test is related to the OC policy-forwarding test for GRE decap), please add a link to PF-1.3 for the required IPoGRE decap configuration rules.